extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -416,6 +413,38 @@ private function getDirTree( $dir ) {
 	}
 
 	/**
+	 * Remove DFS bucket of a given wiki
+	 *
+	 * @see PLATFORM-1700
+	 * @param int $cityId
+	 */
+	private function removeBucket( $cityid ) {
+		try {
+			$swift = \Wikia\SwiftStorage::newFromWiki( $cityid );
+			$this->log( sprintf( "Removing DFS bucket /%s%s", $swift->getContainerName(), $swift->getPathPrefix() ) );
+
+			// s3cmd --recursive del s3://BUCKET/OBJECT / Recursively delete files from bucket
+			$cmd = sprintf(
+				'sudo /usr/bin/s3cmd -c %s --recursive del s3://%s%s/',
```

Comment:
True, true :) CloudFiles was removed in the MediaWiki upstream repo (it's quite hacky and ugly code). WMF guys implemented their own Swift client in PHP.